tests/e2e/script_test.go

```diff
@@ -119,6 +135,23 @@ func TestE2EScript(t *testing.T) {
 						// continue to export the resource
 						shouldGetKubeObject = false
 
+					case "CHECK-INTENT":
```

Comment:
Nit: I don't know what CHECK-INTENT does - is there a version of this name that is more self-documenting?

Reply:
Reworked this and added an entry in the README